apps/dashboard/app/controllers/api/api_sessions_controller.rb

```diff
@@ -0,0 +1,142 @@
+class Api::ApiSessionsController < ApiController
+
+  def index
+    @sessions = BatchConnect::Session.all
+    @sessions.each(&:update_cache_completed!)
+
+    response = @sessions.map do |session|
+      create_session_data session
+    end
+
+    render json: { items: response }
+
+  rescue Exception => error
+    logger.error "action=getSessions user=#{@user} error=#{error}"
```

Comment:
All the `user={@user}` stuff here isn't really relevant. All these logs get spit into `ondemand-nginx/$USER/error.log` anyhow, so the logfile itself is _only_ for that user.

Are you all using a format to ingest these logs?  I guess that's also an issue, this now has a format that none of the other log entries do.

Reply:
This is an Splunk friendly log entry, which is not really relevant for this project.

If you point me to a good example log entry (or tell me what the format should be), I will make a new PR updating these log entries.

Reply:
I have updated the logging entries based on other logging statements found in the Dashboard application.

If you want me to follow another format, please let me know.

Reply:
Yea unfortunately that's kind of the problem, there we don't really have well formatted logs.  We don't index those logs here at OSC and I'm unaware of folks who do.  But clearly, there's a need to standardize formats (or get a logger that can do that for us), we just don't have that yet.  What you have works for me.

Reply:
If we did ingest these logs you'd have to wrap anything with space or quotes in quotes so like `error=\"#{error}\"`.  It's probably safe to assume usernames don't have spaces but also doesn't hurt to wrap everything in quotes.  If there are spaces without quotes then Splunk `key=value` parsing won't work.